### PR TITLE
[presubmit] Add options for verbose logging and running on all files.

### PR DESCRIPTION
**Pull request by jonathanmetzman**
Also fix some issues it found.
And skip static checks when all files is used on files in ignored directories, we're unlikely to fix 
the changes that have been landed, but still good to warn when they are changed.

### CODE REVIEW

File: .github/workflows/fuzzers.yml [hunk 1]
```diff
@@ -2,10 +2,10 @@ name: Build fuzzers
 on:
   pull_request:
     paths:
-      - 'docker/**'  # Base images changes.
+      - 'docker/**'  # Base images image changes.
```
Comment (inferno-chromium) [inline .github/workflows/fuzzers.yml:5]: Just Base image changes ?

File: .github/workflows/fuzzers.yml [hunk 2]
```diff
@@ -2,10 +2,10 @@ name: Build fuzzers
...
       - 'fuzzers/**' # Changes to fuzzers themselves.
       - 'benchmarks/**'  # Changes to benchmarks.
-      - 'src_analysis/**' # Changes that affect what gets built.
+      - 'src_analysis/**' # Changes that effect what gets built.
```
Comment (inferno-chromium) [inline .github/workflows/fuzzers.yml:8]: affect seems right ?

File: presubmit.py [hunk 3]
```diff
@@ -293,21 +317,38 @@ def bool_to_returncode(success: bool) -> int:
...
     os.chdir(_SRC_ROOT)
-    changed_files = [Path(path) for path in diff_utils.get_changed_files()]
+
+    if not args.all_files:
+        relevant_files = [Path(path) for path in diff_utils.get_changed_files()]
+    else:
+        relevant_files = get_all_checkable_files()
+
+    logs.debug('Running presubmit check(s) on: %s',
+               ' '.join(str(path) for path in relevant_files))
```
Comment (inferno-chromium) [inline presubmit.py:348]: maybe '\n' to print paths on each line.

File: third_party/sancov.py [hunk 4]
```diff
@@ -245,13 +245,11 @@ def PrintMissing(binary):
     if not file_list:
         Usage()
 
-    if sys.argv[1] == "print":
-        PrintFiles(file_list)
-    elif sys.argv[1] == "merge":
+    if sys.argv[1] == "merge":
```
Comment (inferno-chromium) [inline third_party/sancov.py:248]: Does the message at start of file needs changing. I thought we use PrintFiles function.

File: presubmit.py [hunk 5]
```diff
@@ -293,21 +317,38 @@ def bool_to_returncode(success: bool) -> int:
...
+    parser.add_argument('--all-files',
+                        action='store_true',
+                        help='Run presubmit check(s) on all files',
+                        default=False)
+    parser.add_argument('-v', '--verbose', action='store_true', default=False)
 
     args = parser.parse_args()
+    if args.verbose:
+        logs.initialize(log_level=logging.DEBUG)
     os.chdir(_SRC_ROOT)
```
Comment (inferno-chromium) [inline presubmit.py:340]: probably move os.chdir before the logs init.

File: .github/workflows/fuzzers.yml (hunk 1 above)
Comment (jonathanmetzman) [reply .github/workflows/fuzzers.yml]: Sorry accidental change.

File: .github/workflows/fuzzers.yml (hunk 2 above)
Comment (jonathanmetzman) [reply .github/workflows/fuzzers.yml]: Yup, don't know how I've somehow unlearned the difference lately.

File: third_party/sancov.py (hunk 4 above)
Comment (jonathanmetzman) [reply third_party/sancov.py:249]: Done.

File: presubmit.py (hunk 5 above)
Comment (jonathanmetzman) [reply presubmit.py:338]: Done.

**Pull request merged**